### PR TITLE
update: Add info about package updates

### PR DESCRIPTION
**Pull request by anselmolsm**

### CODE REVIEW

File: docs/getting-started/update.md [hunk 1]
```diff
@@ -268,3 +268,45 @@ Flight stack developers may want to flash the current version being developed
...
 recommended to follow the steps from each of them: [PX4](https://dev.px4.io/en/flight_controller/intel_aero.html)
 and [ArduPilot](http://ardupilot.org/copter/docs/common-intel-aero-rtf.html).
+
+## Updating packages
+
+The release 1.5 enabled by default the possibility of updates per package, featuring `dnf` as the package management tool.
+
+The main purpose of the official package repository is to make updates of key components available in a faster pace than the OS release cycles. Notice it does not intend to be a fully featured repository as the ones offered by standard Linux distributions - at first, it provides the rpms of packages that are in the .iso, but eventually new ones can be included - For suggestions about packages addition/update/modification,  please open an issue at https://github.com/intel-aero/meta-intel-aero/issues .
+
+First step, required only once: retrive the public
```
Comment (lucasdemarchi) [inline docs/getting-started/update.md]: Retrieve

File: docs/getting-started/update.md [hunk 2]
```diff
@@ -268,3 +268,45 @@ Flight stack developers may want to flash the current version being developed
 rather than the stable version that comes with the OS image. For that it's
 recommended to follow the steps from each of them: [PX4](https://dev.px4.io/en/flight_controller/intel_aero.html)
 and [ArduPilot](http://ardupilot.org/copter/docs/common-intel-aero-rtf.html).
+
+## Updating packages
+
+The release 1.5 enabled by default the possibility of updates per package, featuring `dnf` as the package management tool.
+
+The main purpose of the official package repository is to make updates of key components available in a faster pace than the OS release cycles. Notice it does not intend to be a fully featured repository as the ones offered by standard Linux distributions - at first, it provides the rpms of packages that are in the .iso, but eventually new ones can be included - For suggestions about packages addition/update/modification,  please open an issue at https://github.com/intel-aero/meta-intel-aero/issues .
```
Comment (lucasdemarchi) [inline docs/getting-started/update.md]: "Notice:"? Or "Note that"

**Comment by lucasdemarchi** [comment]
Som comments above, otherwise lgtm

**Comment by anselmolsm** [comment]
Merging with the requested changes.

**Pull request merged**